cf_xarray/tests/test_accessor.py

```diff
@@ -152,7 +153,7 @@ def test_wrapped_classes(obj, attr, xrkwargs, cfkwargs):
         actual = getattr(obj.cf, attr)(**cfkwargs).mean(*args)
     assert_identical(expected, actual)
 
-    if attr in ("groupby", "groupby_bins"):
+    if attr in ("groupby", "groupby_time_accessor", "groupby_bins"):
```

Comment:
```suggestion
    if attr in ("groupby", "groupby_bins"):
```

Reply:
I have not made this change yet, as I'm a little confused by one comment adding "groupby_time_accessor" and one removing it. Currently it IS IN the code and it is passing.

Reply:
yes we should delete it because it isn't doing anything.